docker_yaqluator.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/env sh
```

Comment:
I have tested the script and it works nicely with a small tweak.

One issue I encountered on my macOS is:

```
./docker_yaqluator.sh 
zsh: ./docker_yaqluator.sh: bad interpreter: /bin/env: no such file or directory
```

Can you please replace the shebang line to something that is more portable (like here - https://stackoverflow.com/a/52860837/165927)?

Reply:
Update complete. In addition, per suggestion from a coworker, I added `.env` file with default value for `YAQLUATOR_HTTP_PORT` as example for how to set it.